tools/scripts/test.pkgs.minified.sh

```diff
@@ -0,0 +1 @@
+for pkg in core jquery jquery-file-upload; do node_modules/.bin/karma start --single-run --cloudinary.minified --cloudinary.pkg=${pkg}; done
```

Comment:
@patrick-tolosa , why not "do karma start..."?

Reply:
Right, everything was written as node_modules/.bin/karma but I forgot to rename this one in this branch, I updated the PR.